filebeat/module/auditd/log/test/audit-rhel7.log

```diff
@@ -2445,3 +2445,4 @@ type=DAEMON_END msg=audit(1481078697.892:7799): auditd normal halt, sending auid
 type=CWD msg=audit(1489639811.480:451):  cwd="/home/some_user"
 type=PATH msg=audit(1489639811.480:451): item=0 name="/etc/ssh/sshd_config" inode=34485109 dev=08:01 mode=0100600 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:etc_t:s0 objtype=NORMAL
 type=USER_CMD msg=audit(1489639825.595:452): pid=1325 uid=1000 auid=1000 ses=1 subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 msg='cwd="/home/some_user" cmd=7461696C202D3230202F7661722F6C6F672F61756469742F61756469742E6C6F67 terminal=pts/0 res=success'
+type=ANOM_ABEND msg=audit(1605431420.026:123): auid=12345 uid=123 gid=123 ses=123456789 pid=1234 comm="extproc" reason="memory violation" sig=6
```

Comment:
Only the first 100 events are output to the golden file IIUC test_modules.py correctly. So if you want this to show up in a golden file then I recommend to move it to its own file.